Remotion/Validation/Core/Attributes/Validation/LengthValidationAttribute.cs

```diff
@@ -40,39 +40,56 @@ public class LengthValidationAttribute : AddingValidationAttributeBase
     /// <param name="maxLength">The maximum number of characters allowed.</param>
     public LengthValidationAttribute (int minLength, int maxLength)
```

Comment:
In the ticket it says that the constructor has to be parameterless. Is this ctor meant? Or the ones from the ExactLength & MinLength Validators?

Reply:
Ah, the ticket has to be updated. It is not possible to make this ctor parameterless, because the idea was at first to set the properties using a setter, but this doesn't work with attributes. That's why, we have to keep it like this.

Reply:
There was a misunderstanding: we want the parameterless ctor and use zero instead of null as the default for the properties since nullable integers are not supported on attributes.